zowe/zowe-cli
# fix(secrets): Add `prebuildCheck.js` script to check if prebuilds exist when building

**What It Does**

This PR adds a `preinstall.js` script to the Secrets SDK install process so that it will only fail if the prebuilds are not found during installation, triggering a rebuild.

Fixes #1787 - if the user's architecture/platform is not supported, Cargo is still required to build the Secrets SDK.

**How to Test**

First run the `prepublishOnly` script for the Secrets SDK so that it downloads the prebuilds locally. Then, try installing Zowe CLI with this branch in a Linux environment without `libsecret` installed. The CLI should still install without throwing any errors with the Secret SDK during installation.

**Review Checklist**
I certify that I have:
